lang/java/reef-webserver/src/main/java/org/apache/reef/webserver/HttpServerReefEventHandler.java

```diff
@@ -189,8 +193,9 @@ public void onHttpRequest(
         }
       }
       break;
-    // TODO[JIRA REEF-798] Use this provider in the HTTP
     case "progress":
+      response.getWriter().println(progressProvider.getProgress());
```

Comment:
Good question. As this essentially just presents the information we give to YARN, a single number seems fine. If we ever wanted a more elaborate progress schema, we could create another REST endpoint for that.